test/common/integration/base_client_integration_test.cc

```diff
@@ -61,6 +63,10 @@ BaseClientIntegrationTest::BaseClientIntegrationTest(Network::Address::IpVersion
   use_lds_ = false;
   autonomous_upstream_ = true;
   defer_listener_finalization_ = true;
+
+  if (auto log_level = TestEnvironment::getOptionalEnvVar("LOG_LEVEL")) {
+    addLogLevel(Platform::logLevelFromString(*log_level));
```

Comment:
can we skip the env variable and set this from TestEnvironment::getOptions().logLevel()  ?  Not sure if it'll work so check please :-)
(sorry didn't comment that earlier, forgot where it lived)

Reply:
Good point! Done, and it works :)